CHANGELOG.md

```diff
@@ -1,3 +1,7 @@
+# 3.5.0 (2019-09-27)
+
+- Added support for `@skyux-sdk/builder@3.8.0`. [#49](https://github.com/blackbaud/skyux-list-builder-view-grids/pull/49)
```

Comment:
Should this be "Updated" and a patch update? We have a precedent for this elsewhere: https://github.com/blackbaud/skyux-modals/blob/master/CHANGELOG.md#306-2019-07-05

Reply:
Maybe? We've used `Added` in other places too but it was bundled with other things. I would vote for a minor since we also bumped some peer dependencies. Thoughts?

Reply:
Oh yeah, I guess we haven't been very consistent. I see it the other way in a few places. Ok, lets ship it.